src/indieweb_utils/replies/context.py

```diff
@@ -29,298 +40,316 @@ def get_reply_context(url, twitter_bearer_token=None):
     :rtype: list
     """
 
-    h_entry = None
-    photo_url = None
-    site_supports_webmention = False
+    return_object = ReplyContext
+    photo_url = ""
     parsed_url = url_parse.urlsplit(url)
     http_headers = {"Accept": "text/html", "User-Agent": "indieweb_utils"}
 
-    if url.startswith("https://") or url.startswith("http://"):
-        try:
-            page_content = requests.get(url, timeout=10, verify=False, headers=http_headers)
-        except:
-            return True, {"error": "Page content could not be retrieved."}
+    if not url.startswith("https://") and not url.startswith("http://"):
```

Comment:
Since we have the parsed url already split, we could simplify this.

```suggestion
    if parsed_url.scheme not in ["http", https"]:
```